spec/defines/fmw_spec.rb

```diff
@@ -69,11 +69,11 @@
                   :source               => '/mnt',
                   :temp_directory       => '/tmp',
                 }}
-    let(:title) {'My Oracle-"SOA" Installation!?'}
```

Comment:
This spec test was added in https://github.com/biemond/biemond-orawls/commit/d75151ea03e571c767c0fb3c900314ebc5b392cd

The resource title was interesting on purpose. :)
The context is 'resource title needing sanitising'
https://github.com/biemond/biemond-orawls/blob/master/spec/defines/fmw_spec.rb#L56